### PR TITLE
Adding option to remove previously read topics from new conversations tab

### PR DESCRIPTION
**Pull request by Caseysch**
Makes it easier to keep up with yammer if checked infrequently. Left unchecked by default because it can be a little jarring.

### CODE REVIEW

**Comment by jessey-git** [comment]
Looks good and seems to work on my end, but yeah, it's hard to follow unless you know it's going on.  I'll merge in tonight if you don't get to it before me.

File: scripts/yamblock.js [hunk 1]
```diff
@@ -50,7 +54,8 @@ function processDOMChanged() {
             posts: {
                 hideGroups: true,
                 hideJoins: true,
-                hidePraises: true
```
Comment (jessey-git) [inline scripts/yamblock.js]: I'm open to ideas on how to remove this particular duplication of 'default' values for storage. Any ideas how to share the json?

Comment (Caseysch) [inline scripts/yamblock.js]: Looks like it can skip the defaults here and be replaced by chrome.storage.sync.get({ users, posts, options }, function (data)...

Comment (jessey-git) [inline scripts/yamblock.js]: Hmm, I'll give that a try.  Thanks for the idea.

Comment (Caseysch) [inline scripts/yamblock.js]: nevermind, looks like I didn't reload the extension or something like that. Doesn't work >.<
